pages/tutorials/index.html

```diff
@@ -33,52 +37,58 @@ <h1 class="display-4">Getting started with Cantera</h1>
-          <a href="../matlab/tutorial.html" class="btn btn-primary">Matlab Tutorial</a>
+          <a href="MatlabTutorial.html" class="btn btn-primary">Matlab Tutorial</a>
         </div>
       </div>
     </div>
   </div>
 </section>
 <br />
-<section id="cantera-tutorials">
+<section id="cantera-next-steps">
   <h2 class="display-5">Next steps</h2>
   <div class="jumbotron">
 
-  <p class="lead">Okay, so you have finished the tutorials and understand the basic user functionality
-  of Cantera. Now what?
-</p>
-<p> Using Cantera for a range of problems will likely require you to extend your
-  knowledge in two ways:
-  <ul>
-    <li> You will need an input file describing the phase(s) of matter relevant to your problem.
-    </li>
-    <li> Your application may very well require function calls and routines not necessarily covered
-      in the tutorials.
-    </li>
-  </ul>
+  <p class="lead">Okay, so you have finished the tutorials and understand the
+  basic user functionality of Cantera. Now what?
+  </p>
+  <p> Using Cantera for a range of problems will likely require you to extend
+    your knowledge in two ways:
+    <ul>
+      <li> You will need an input file describing the phase(s) of matter
+        relevant to your problem.
+      </li>
+      <li> Your application may very well require function calls and routines
+        not necessarily covered in the tutorials.
+      </li>
+    </ul>
   </p>
   <p> The links below will help you take the 'next steps,' and point you to:
     <ul>
-      <li> Information on how to locate and work with Cantera input files (which contain the thermodynamic, transport,
-        and chemical kinetic information for the phases of interest).
+      <li> Information on how to locate and work with Cantera input files (which
+        contain the thermodynamic, transport, and chemical kinetic information
+        for the phases of interest).
       </li>
-      <li> Detailed documentation ans user guides for accessing Cantera via Python, Matlab, and directly
-        via C++. For advanced and intermediate users, the documentation is an easily-searchable repository
-        for information on specific functions of interest.
+      <li> Detailed documentation ans user guides for accessing Cantera via
+        Python, Matlab, and directly via C++. For advanced and intermediate
+        users, the documentation is an easily-searchable repository for
+        information on specific functions of interest.
       </li>
-      <li> A repository of examples, demonstrating how to use Cantera to solve a diverse range of problems.
-        You can either use these examples directly, or use them as a template to develop your own applications.
+      <li> A repository of examples, demonstrating how to use Cantera to solve a
+        diverse range of problems. You can either use these examples directly,
+        or use them as a template to develop your own applications.
       </li>
     </ul>
   </p>
```

Comment:
I can't comment on the line for `InputFiles`, but for that one you have the slug command set properly, so the file is `input-tutorials.html`, and you'll want to drop the leading `/`, which points to a file in the root of the website.

Reply:
Okay, I think I got these cleaned up.